nevergrad/functions/games/game.py

```diff
@@ -0,0 +1,423 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Discussions with Tristan Cazenave, Bruno Bouzy, have been helpful.
+# Dagstuhl's 2019 seminar on games has been helpful (seminar 19511).
+
+from typing import Any
+from typing import List
+#import matplotlib.pyplot as plt
+from ... import instrumentation as inst
+from ...instrumentation.multivariables import Instrumentation
+import numpy as np
+
+
+class _Game(object):
+    def __init__(self):
+        self.verbose = False
+        self.history1 = []
+        self.history2 = []
+        self.batawaf = False
+        self.converter = {
+                "flip": lambda p1, p2: self.flip_play_game(p1, p2),
```

Comment:
please add tests for each game (just loop over them maybe)

Reply:
added test_games.py